core/server/data/export/index.js

```diff
@@ -21,12 +21,14 @@ exportFileName = function exportFileName(options) {
     var datetime = (new Date()).toJSON().substring(0, 10),
         title = '';
 
+    options = options || {};
+
     return models.Settings.findOne({key: 'title'}, _.merge({}, modelOptions, options)).then(function (result) {
         if (result) {
             title = security.string.safe(result.get('value')) + '.';
         }
 
-        return title + 'ghost.' + datetime + '.json';
+        return options.filename ? options.filename + '.json' : title + 'ghost.' + datetime + '.json';
```

Comment:
If we pass a custom filename, no need to request the settings table.

Reply:
@kirrg001 Sorry only just seen this.
The default export when no filename is provided still uses the title from settings

Reply:
Yeah i know, but if you pass `options.filename`, there is no need to query the db? So the condition can be moved to the top of the function like:

```
// custom filename
if (options.filename) {
  return Promise.resolve(options.filename + '.json');
}

// default, blog title
return models.Settings.findOne(...)
```

Reply:
Ahh I see what you mean now. Will update the PR